jmaxml-wasm/src/lib.rs

```diff
@@ -4,7 +4,7 @@ use wasm_bindgen::prelude::*;
 
 #[wasm_bindgen(js_name = parseXml)]
 pub fn parse_xml(content: &str) -> Result<JsValue, String> {
-    let report = match jmaxml::from_str(content) {
+    let report = match jmaxml::Report::new(content) {
```

Comment:
I think the `new` typically implies creating a new, fresh instance rather than parsing existing XML content. `Report::from_str` (https://doc.rust-lang.org/std/str/trait.FromStr.html) would follow conventions for parsing from strings.